src/sql/plugins/appointment/plugin/create_db_appointment.sql

```diff
@@ -64,6 +64,8 @@ CREATE TABLE IF NOT EXISTS appointment_form (
   is_multislot_appointment BOOLEAN DEFAULT FALSE NOT NULL,
   role_fo varchar(255),
   capacity_per_slot INT DEFAULT 0 NOT NULL,
+  nb_consecutive_slots INT DEFAULT 0 NOT NULL,
+  label_consecutive_slots VARCHAR(255) NULL,

```

Comment:
Why storing a label in the database, how to internalize this label?

Reply:
Hello,

This label was added so each form can have its own custom name for the "_Consecutive slots_" field (previously shown as "_Nombre de personnes_" or "_Number of people_" to the user). As per the Redmine issue's requirements:

> The field label must be configurable

So, instead of a defined list of label names to pick from, we let the form's creator choose a name, which requires to be saved in the database.